setup.cfg

```diff
@@ -1,12 +1,13 @@
 [flake8]
-# These are exceptions allowed (encouraged?) by Galaxy style guidelines.
+# These are exceptions allowed by Galaxy style guidelines.
 # 128 continuation line under-indented for visual indent
 # 201 and 202 are spaces after ( and before )
 # 203 whitespace before ':'
 # 402 module level import not at top of file # TODO, we would like to improve this.
 # 501 is line length
 # W503 is line breaks before binary operators, which has been reversed in PEP 8.
-ignore = E128,E201,E202,E203,E501,E402,W503
+# D** are docstring linting - which we mostly ignore except D302. (Hopefully we will solve more over time).
+ignore = E128,E201,E202,E203,E501,E402,W503,D100,D101,D102,D103,D104,D105,D200,D201,D202,D204,D205,D206,D207,D208,D209,D210,D211,D300,D301,D400,D401,D402,D403
```

Comment:
The doc it's indeed a bit confusing, but the proposed syntax works (it's actually the same you use in `.ci/flake8_wrapper_docstrings.sh`). My idea was to prevent new errors/warnings when a new plugin release will add a new code, but again no strong opinion.